documentation/Documentation/Objectives-List.md

```diff
@@ -429,6 +429,33 @@ This objective can be completed by riding the specified
     ride any
     ```
 
+## Run a Command: `command`
+
+To complete this objective the player must execute a specified command. The first argument is the command string. Use
+`_` in place of spaces for the command. The command argument is case-sensitive and also supports using placeholders. The
+second required argument is a list of events to execute when the objective is met.
+
+!!! example
+    ```YAML
+    command /warp_%player%_farms events:event1,event2
+    ```
+
+With this configuration, the command objective requires the player to execute `/warp MyName farms` to be completed. The
+command objective matches from the start of the command that was executed, therefore if the player executed
+`/warp MyName farms other arguments` it would still be completed.
+
+Optional arguments:
+* `ignoreCase`: If provided, instructs the objective to ignore case for the command.
+* `exact`: If provided, requires an exact command match, not just the command start.
+* `cancel`: If provided, the objective will cancel the execution of the command on a match.
+* `conditions`: If provided, specifies a list of conditions required to meet the objective in addition to the command.
+* `failEvents`: If provided, specifies a list of events to execute if a non-matching command is run and conditions are met.
```

Comment:
It it correct, that this is not the implementation of my mentioned `cancelEvents`?

I think this is a new not mentioned feature. And i like it. Only thing:
I think if you don't meet the command, and the `failEvents` are executed, you may also cancel the command, that was not the target command. For that we would need one more argument like `cancel` but with a name like `failCancel`.

Reply:
This is trying to make sense of the earlier request. After working through the documentation, I could not figure out a use case for cancelEvents. If the cancel flag is set, the cancel events would be the same events already defined.
In regard to the cancel conditions, I still could not figure out a use case for it. Conditions apply to the objective itself, and would govern if the command is cancelled or not already. Both of these are redundant and not needed. Please help me understand if there is a use case that I'm missing.
For failCancel, I'm not sure that we should allow the objective to cancel events that are run that are not related to the objective. This might prevent people from switching to a minigame, exiting, and other things. I don't think we should do it.